modules/ztp-deploying-a-site.adoc

```diff
@@ -116,7 +116,7 @@ spec:
     nodes:
       - hostName: "example-node1.example.com" <6>
         role: "master"
-        bmcAddress: "idrac-virtualmedia+https://[1111:2222:3333:4444::bbbb:1]/redfish/v1/Systems/System.Embedded.1" <7>
+        bmcAddress: idrac-virtualmedia://<out-of-band-ip>/<systemID>/ <7>
```

Comment:
```suggestion
        bmcAddress: idrac-virtualmedia://<out_of_band_ip>/<system_id>/ <7>
```
See the [repo guidelines on replaceable values](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) (you need to scroll a bit).

Reply:
Fixed, thanks @michaelryanpeter - happy Friday!

Reply:
Excellent! Could you confrim you are ready for a merge by adding the merge review needed label?